### PR TITLE
README change

### PR DESCRIPTION
**Pull request by keithr25**
I added some text to the README file just to practice pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,4 +24,5 @@ Your task is to study how the user CRUD was built and to develop another entity
 
 You'll also need to make your own special user interface. The existing one is pretty lame. Create that separately from the Backbone code and when it's looking nice, introduce it into the code base.
 
-Have fun.
\ No newline at end of file
+Have fun.
+ADDING SOME TEXT TO PRACTICE PULL REQUEST
```
Comment (keithr25) [inline README.md]: Looks good

**Comment by keithr25** [comment]
I downloaded the robo images in the db.json file and put them in a images folder

**Comment by Hector-Ivan** [comment]
Brad isn't here, he may be in later this afternoon i think, but this morning Kevin showed us a desktop application we can use for git instead of working from the command line, he said it was our choice... we all downloaded it, it's on GitHub Desktop
|   |
|   |  |   |   |   |   |   |
| GitHub DesktopSimple collaboration from your desktop |
|  |
| View on desktop.github.com | Preview by Yahoo |
|  |
|   |

**Comment by keithr25** [comment]
ok

**Pull request closed without merging**